api/models/enrollment_tests.py

```diff
@@ -0,0 +1,50 @@
+from django.db import models
+
+from api.models.constants import DEFAULT_CHAR_FIELD_MAX_LEN
+from api.models.languages_levels import TeachingLanguageAndLevel
+from api.models.people import Student
+
+
+class EnrollmentTest(models.Model):
+    """Model for 'written' test given to the student at registration."""
+
+    languages_and_levels = models.ManyToManyField(TeachingLanguageAndLevel)
+
+    def __str__(self):
+        return ", ".join(self.languages_and_levels.all())
+
+
+class EnrollmentTestQuestion(models.Model):
+    enrollment_test = models.ForeignKey(EnrollmentTest, on_delete=models.CASCADE)
+    text = models.CharField(max_length=DEFAULT_CHAR_FIELD_MAX_LEN, unique=True)
```

Comment:
Are you sure that it should be unique?
For example, what if we want to have same question for different levels? 
I recommend to do the same thing as in EnrollmentTestQuestionOption - unique pair with text+test

Reply:
hmm... yes, let's do that to be on the safe side